### PR TITLE
restructure docs

### PR DESCRIPTION
**Pull request by laisbsc**
This substantial PR addresses the following documentation changes:
- [x] Deletion of unnecessary files on the tree. 🟥 
- [x] Update the structure of the documentation to one more modular. 🔬 
- [x] Edit content for the [introduction](https://github.com/Quansight/qhub-cloud/blob/cf973386a56551c5e3726e369f1ed74d42646fc3/docs/source/1_intro/1_intro.md) - kept the same, just slightly trimmed. ✂️ 
- [x] Create a Get Started section containing:
    - [x] [installation](https://github.com/Quansight/qhub-cloud/blob/79a2f5be11f5eb64af63d05cdbc7cc4de13d6d9b/docs/source/2_get_started/1_installation.md) 📨 
    - [x] [setup](https://github.com/Quansight/qhub-cloud/blob/79a2f5be11f5eb64af63d05cdbc7cc4de13d6d9b/docs/source/2_get_started/2_setup.md) ✍🏽 
- [x] Submit PR for review. 🧑‍💻

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -1,28 +1,55 @@
...
-)  # autoapi.extension
-exclude_patterns = ["_build", "*checkpoint*", "output", "outputs"]
+
+extensions = [
+    "myst_parser",
+    "nbsphinx",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.napoleon",
+    "sphinx_copybutton",
+    # .split()
```
Comment (costrouc) [inline docs/conf.py:36]: Can remove this line `# .split()`

**Comment by costrouc** [comment]
Awesome work @laisbsc this is looking like great progress to me.

**Comment by costrouc** [comment]
@laisbsc I changed the branch the commit goes into `dev`

image

This did cause some merge conflicts. You'll need to make sure in the future to always branch off of `dev` and not `main` otherwise there will be these merge conflicts. We will probably need to add contribution docs around specifically how to create a PR since we will not be following the typical github flow.

**Comment by laisbsc** [comment]
> @laisbsc I changed the branch the commit goes into `dev`

Cool. I thought I had changed the branch to merge. Sorry for the distraction and thank you for fixing it.

> This did cause some merge conflicts.

I will address them now.

> You'll need to make sure in the future to always branch off of `dev` and not `main` otherwise there will be these merge conflicts. 

Noted.

> We will probably need to add contribution docs around specifically how to create a PR since we will not be following the typical GitHub flow.

This is the next step. Adding contribution guidelines for QHub on code (specifying tests and best practices) and docs (where to deploy and formatting conventions).
Thanks for the help!

File: setup.py [hunk 2]
```diff
@@ -19,9 +19,9 @@
     url="https://github.com/quansight/qhub",
     author="Quansight",
     classifiers=[
-        "Development Status :: 3 - Alpha",
+        "07_development Status :: 3 - Alpha",
```
Comment (costrouc) [inline setup.py:22]: Was this intentional? Looks like a find a replace issue

Comment (laisbsc) [reply setup.py:22]: Nope! I did not touch that. Some of the config I couldn't make sense of and left just as it was.

**Comment by costrouc** [comment]
@laisbsc this should have been merged until we had time to check that CI passed. The flake8 errors are due to indentation of the strings in setup.py.

**Comment by costrouc** [comment]
Resolved issue with https://github.com/Quansight/qhub-cloud/pull/330

**Pull request merged**